GitHubCard/index.js

```diff
@@ -45,9 +45,9 @@ axios.get('https://api.github.com/users/uchxchi')
 
 const followersArray = ['kip-guile', 'coremand', 'dmedunoye', 'justsml', ' bigknell'];
 const getData = followersArray.forEach(str => {
-  // axios.get('https://api.github.com/users/')
+  axios.get('https://api.github.com/users/') //something like this? 
 })
-console.log(getData)
+
```

Comment:
Yea I think you are on the right track. you can try something like this. See if it works.

```
  axios.get(`https://api.github.com/users/${str}`)
.then(response => {
     document.querySelector('.cards').appendChild(myGitCard(response.data));
})
.catch(err => console.error(err))
```